tests/support/e2e.ts

```diff
@@ -20,3 +20,9 @@ import { DataverseApiAuthMechanism } from '@iqss/dataverse-client-javascript/dis
 
 const VITE_DATAVERSE_BACKEND_URL = (import.meta.env.VITE_DATAVERSE_BACKEND_URL as string) ?? ''
 ApiConfig.init(`${VITE_DATAVERSE_BACKEND_URL}/api/v1`, DataverseApiAuthMechanism.SESSION_COOKIE)
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import registerCypressGrep from '@cypress/grep'
+// eslint-disable-next-line @typescript-eslint/no-unsafe-call
+registerCypressGrep()
```

Comment:
I couldn't find where the grep library is being used, is it to use the cy.wait(500) command in the integration test?

Reply:
I used it from the command line when I was testing locally. For example, 
` npx cypress run --spec tests/e2e-integration/e2e/sections/dataset/Dataset.spec.tsx --browser chrome --env grep="loads the restricted files when the user is logged in as owner",burn=10`

Even though it's not a part of the automated test, I thought it would be good to have in the project, to make it easier to run tests like this locally, for timing errors.  Does that sound ok, or should I keep it part of my local environment?

Reply:
Sounds good! We could document this command in the [Dataverse Frontend Guide](https://github.com/IQSS/dataverse-frontend/issues/26) so that everyone is aware that tests can be run with this command